lib/go/thrift/simple_server.go

```diff
@@ -48,15 +49,26 @@ var ErrAbandonRequest = errors.New("request abandoned")
 // If it's changed to <=0, the feature will be disabled.
 var ServerConnectivityCheckInterval = time.Millisecond * 5
 
+// ServerCloseTimeout defines max close wait duration used by
+// server stop to avoid hanging too long to wait for all client connections to be closed gracefully.
+//
+// It's defined as a variable instead of constant, so that thrift server
+// implementations can change its value to control the behavior.
+//
+// If it's changed to <=0, the feature will be disabled, and the server will wait for
+// for all the client connections to be closed gracefully.
+var ServerCloseTimeout = time.Duration(0)
```

Comment:
On second thought a better name is probably `ServerStopTimeout`, since the function using it is `Stop`, not `Close`.